src/containers/TriggerBinding/TriggerBinding.js

```diff
@@ -0,0 +1,308 @@
+      ({ name, value }) => ({
+        id: name,
+        name,
+        value
+      })
+    );
+
+    const emptyTextMessage = intl.formatMessage({
+      id: 'dashboard.triggerBinding.noParams',
+      defaultMessage: 'No parameters found for this TriggerBinding.'
+    });
+
+    return (
+      <div className="triggerBinding">
+        <h1>{triggerBindingName}</h1>
+        <Tabs
+          onSelectionChange={index => {
+            this.handleTabSelect(index);
+          }}
+        >
+          <Tab
+            label={intl.formatMessage({
+              id: 'dashboard.triggerBinding.details',
+              defaultMessage: 'Details'
+            })}
+          />
+          <Tab label="YAML" />
+        </Tabs>
+        {showDetails && (
+          <div className="details">
+            <p>
+              <span>
+                {intl.formatMessage({
+                  id: 'dashboard.triggerBinding.dateCreated',
+                  defaultMessage: 'Date Created:'
+                })}
+              </span>
+              <FormattedDate
+                date={triggerBinding.metadata.creationTimestamp}
+                relative
+              />
+            </p>
+            <p>
+              <span>
+                {intl.formatMessage({
+                  id: 'dashboard.triggerBinding.labels',
+                  defaultMessage: 'Labels:'
+                })}
+              </span>
+              {formattedLabelsToRender.length === 0
+                ? intl.formatMessage({
+                    id: 'dashboard.triggerBinding.noLabels',
+                    defaultMessage: 'None'
+                  })
+                : formattedLabelsToRender.map(label => (
+                    <Tag
+                      type="blue"
+                      key={`${triggerBindingName}:label:${label}`}
+                    >
+                      {label}
+                    </Tag>
+                  ))}
+            </p>
+            <p>
+              <span>
+                {intl.formatMessage({
+                  id: 'dashboard.triggerBinding.namespace',
+                  defaultMessage: 'Namespace:'
+                })}
+              </span>
+              {triggerBinding.metadata.namespace}
+            </p>
+            <Table
+              title={intl.formatMessage({
+                id: 'dashboard.triggerBinding.parametersTableTitle',
+                defaultMessage: 'Parameters'
+              })}
+              headers={headersForParameters}
+              rows={rowsForParameters}
+              selectedNamespace={selectedNamespace}
+              emptyTextAllNamespaces={emptyTextMessage}
+              emptyTextSelectedNamespace={emptyTextMessage}
+              isSortable
+            />
+          </div>
+        )}
+        {showYAML && (
+          <div className="bx--snippet--multi">
+            <code>
+              <pre>{jsYaml.dump(triggerBinding)}</pre>
+            </code>
+          </div>
```

Comment:
This is going to be duplicated on most resources, it would be good to have this as a simple component that just accepts the resource as a prop

Reply:
agree. Include that in this PR or separate issue?

Reply:
separate is fine, should go in before we end up duplicating this too much though